NoSoft-SA/nspack
# Multi Rebin Verification

Summary of this change:
Multi Rebin Verification
Allows user to verify upto three bins at a time

How to test this code via the UI:
Rmd | Raw Materials | multi_rebin_verification
- Enter number of bins to verify and the total gross_weight
- Enter carton_label_ids to be converted to rebins
